opentelekomcloud/data_source_opentelekomcloud_cce_cluster_v3.go

```diff
@@ -63,6 +64,10 @@ func dataSourceCCEClusterV3() *schema.Resource {
 				Type:     schema.TypeString,
 				Computed: true,
 			},
+			"authentication_mode": {
```

Comment:
in doc this is optional value

Reply:
`Computed` without `Required` or `Optional` means that field can't be set by the user at all. For data source such fields are not part of `ListOpts` but returned in the `Get` response